tests/unit/amo/pages/TestSearchPage.js

```diff
@@ -125,42 +125,6 @@ describe(__filename, () => {
     sinon.assert.callCount(fakeDispatch, 1);
   });
 
-  it('dispatches a server redirect when `atype` parameter is "4"', () => {
-    const fakeDispatch = sinon.spy(store, 'dispatch');
-
-    render({
-      location: createFakeLocation({ query: { atype: 4 } }),
-      store,
-    });
-
-    sinon.assert.calledWith(
-      fakeDispatch,
-      sendServerRedirect({
-        status: 301,
-        url: '/en-US/android/search/?type=search',
-      }),
-    );
-    sinon.assert.callCount(fakeDispatch, 1);
-  });
-
-  it('dispatches a server redirect when `atype` parameter is "5"', () => {
-    const fakeDispatch = sinon.spy(store, 'dispatch');
-
-    render({
-      location: createFakeLocation({ query: { atype: 5 } }),
-      store,
-    });
-
-    sinon.assert.calledWith(
-      fakeDispatch,
-      sendServerRedirect({
-        status: 301,
-        url: '/en-US/android/search/?type=language',
-      }),
-    );
-    sinon.assert.callCount(fakeDispatch, 1);
-  });
```

Comment:
I think we need to keep this test case.

Reply:
I added this test case base as it does not break anything.